routers/api/v1/user/user.go

```diff
@@ -166,3 +166,24 @@ func GetUserHeatmapData(ctx *context.APIContext) {
 	}
 	ctx.JSON(http.StatusOK, heatmap)
 }
+
+// CreateOauth2Application is the handler to create a new OAuth2 Application for the authenticated user
+func CreateOauth2Application(ctx *context.APIContext, data api.CreateOauthApplicationOption) {
```

Comment:
Please add swagger annotations for this handler then run `make generate-swagger`